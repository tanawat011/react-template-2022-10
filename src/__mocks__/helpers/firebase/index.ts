import type * as Firebase from 'helpers/firebase'

const firebase = jest.createMockFromModule<typeof Firebase>('helpers/firebase')

module.exports = firebase
