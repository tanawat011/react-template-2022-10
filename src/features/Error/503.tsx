export const Error503 = () => {
  return <div>503 Service Unavailable</div>
}
