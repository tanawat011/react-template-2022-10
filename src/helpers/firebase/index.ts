export * from './deleteCollection'
export * from './deleteDocument'
export * from './getAllDocuments'
export * from './getDocument'
export * from './getDocumentWithRef'
export * from './setDocument'
export * from './subscribeCollection'
export * from './subscribeDocument'
export * from './updateDocument'
