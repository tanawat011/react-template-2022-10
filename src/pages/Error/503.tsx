import { PATH } from 'Routes'
import { Button } from 'components/Button'

export const Error503 = () => {
  return (
    <div>
      <p>503 Service Unavailable</p>

      <Button link to={`${PATH.BACKOFFICE.ROOT}`}>
        Back to Home Page
      </Button>
    </div>
  )
}
