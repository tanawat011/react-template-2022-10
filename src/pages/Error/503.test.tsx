import { renderWithProviders } from 'helpers/test'

import { Error503 } from './503'

describe('<Error503 />', () => {
  test('match snapshot', () => {
    const { asFragment } = renderWithProviders(<Error503 />)
    expect(asFragment()).toMatchSnapshot()
  })
})
