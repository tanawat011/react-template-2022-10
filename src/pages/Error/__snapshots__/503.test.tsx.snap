// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error503 /> match snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      503 Service Unavailable
    </p>
    <button
      data-testid="button"
      to="/"
    >
      Back to Home Page
    </button>
  </div>
</DocumentFragment>
`;
